### PR TITLE
New package: topgrade-1.0.1

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

File: srcpkgs/topgrade/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'topgrade'
+pkgname=topgrade
+version=1.0.1
+revision=1
+build_style=cargo
+short_desc="Upgrade everything"
```
Comment (maxice8) [inline srcpkgs/topgrade/template]: This is not descriptive

**Comment by jcgruenhage** [comment]
Marked aarch64 as broken, because of nix-rust/nix#951

File: srcpkgs/topgrade/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'topgrade'
+pkgname=topgrade
+version=1.0.1
+revision=1
+build_style=cargo
+short_desc="Meta upgrade tool for pip, flatpak, your distro and everything else"
+maintainer="jcgruenhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/topgrade/template]: GPL-3.0 need the -or-later or -only indentifier

**Comment by jcgruenhage** [comment]
@jnbr I think that is something that should be fixed upstream and not be patched as part of the packaging. Marking it as broken is the better approach IMO.

**Comment by jnbr** [comment]
> @jnbr I think that is something that should be fixed upstream and not be patched as part of the packaging. Marking it as broken is the better approach IMO.

Can to explain why? How is this different to any other musl patch we have?

**Comment by jcgruenhage** [comment]
As I understand this, the problem has been fixed in the nix library upstream, but they haven't done a release in a while (which they probably have reasons for), and this patch just bumps the dependency up to some untagged commit. I think it's better to have it marked as broken and talk to the library maintainers about what they need to do for making a release of their library, and then ask the application maintainer to update to that release.

**Comment by jcgruenhage** [comment]
Okay. Considering this is just a single line changed from 0.11.0, it is safe, yes. I wonder why they didn't tag that as 0.11.1 though.

**Comment by jcgruenhage** [comment]
I've written the patch and am currently trying to build it. If that works, I'll update the PR.

**Comment by maxice8** [comment]
Merged manually after fixing the path in the patches

**Comment by jcgruenhage** [comment]
Thanks. I don't really get how exactly I need to create the patch file for xbps-src to accept it.

**Pull request closed without merging**